examples/basic/sdk/ontology.json

```diff
@@ -598,5 +598,38 @@
       "rid": "query.rid2",
       "version": "0.1.2"
     }
+  },
+  "__UNSTABLE_interfaceTypes": {
```

Comment:
The backend doesnt actually return this right now, so I just hacked something together.